Docs-4188 SCALE 22.02 Updated Periodic Snapshot Task Articles (#1536)
truenas/documentation: content/SCALE/SCALEUIReference/DataProtection/PeriodicSnapshotTasksScreensSCALE.md

---
title: "Periodic Snapshot Tasks Screens"
description: "This article provides information on the data protection periodic snapshot task settings and screens."
weight: 50
aliases:
tags:
 - scalesnapshots
 - scalereplication
---

{{< toc >}}

The **Data Protection** screen **Periodic Snapshot Task** widget displays periodic snapshot tasks created on the system. 
A periodic snapshot task allows scheduling the creation of read only versions of pools and datasets at a given point in time.

## Periodic Snapshot Task Widget

The **Periodic Snapshot Task** widget displays a list of tasks configured on the system.

![PeriodicSnapshotTaskWidget](/images/SCALE/22.02/PeriodicSnapshotTaskWidget.png "Periodic Snapshot Task Widget") 

If a periodic snapshot task is not yet configured **No Periodic Snapshot Task configured** displays in the widget.

![PeriodicSnapshotTaskWidgetNoTasks](/images/SCALE/22.02/PeriodicSnapshotTaskWidgetNoTasks.png "No Periodic Snapshot Tasks")

**Add** opens the **[Add Periodic Snapshot Task]()** screen. 

> Each task listed is a link that opens the **[Edit Periodic Snapshot Task]()** screen populated with with the settings for that task. Click on the **Description**, **Frequency** or **Next Run** column entry to open the edit task screen.

Author: I need to fix that relref that is missing

## Periodic Snapshot Task List Screen
Periodic snapshot tasks display on both the **Data Protection** widget and **Periodic Snapshot Tasks** list screen.
{{< expand "Click Here for More Information" "v" >}}
Click on the **Periodic Snapshot Task** header to open the **Data Protection > Periodic Snapshot Task** list screen.

![DataProtectionPeriodicSnapshotTasksList](/images/SCALE/22.02/DataProtectionPeriodicSnapshotTasksList.png "Periodic Snapshot Task List")

If a task is not added, the list view displays **Add Periodic Snapshot Tasks** which opens the **[Add Periodic Snapshot Task]()** screen. 

![DataProtectPeriodicSnapshotTasksDetailsNoTasks](/images/SCALE/22.02/DataProtectPeriodicSnapshotTasksDetailsNoTasks.png "Periodic Snapshot Task No Tasks")

**Columns** displays a dropdown list of options to customize the list view. Options are **Select All**, **Recursive**, **Naming Schema**, **When**, **Frequency**, **Next Run**, **Keep snapshot for**, **VMWare Sync**, **Enabled**, **State**, and **Reset to Defaults**.

> The **State** on the list view does not link to the log file or anything else, it just displays the current state of the task.

Reviewer: ```suggestion
The **State** on the list view does not link to the log file or anything else. It only displays the current state of the task.
```

> Click the <span class="material-icons">expand_more</span> expand icon at the right of the task opens the details for the selected task.

Reviewer: ```suggestion
Click the <span class="material-icons">expand_more</span> expand icon at the right of the task to open the details for the selected task.
```

![DataProtectionPeriodicSnapshotTasksDetails](/images/SCALE/22.02/DataProtectionPeriodicSnapshotTasksDetails.png "Periodic Snapshot Task Details")

**Edit** opens the **[Edit Periodic Snapshot Task]()** screen.

**Delete** opens the delete dialog that removes the task from the system. 
{{< /expand >}}
## Add and Edit Periodic Snapshot Screens

> The **Add Periodic Snapshot Task** and **Edit Periodic Snapshot Task** display the some of the same settings.

Reviewer: ```suggestion
The **Add Periodic Snapshot Task** and **Edit Periodic Snapshot Task** screens display some of the same settings.
```
